src/driver/main.cpp

```diff
@@ -467,6 +467,10 @@ struct compiler
            {"--exhaustive-tune"},
            ap.help("Exhastively search for best tuning parameters for kernels"),
            ap.set_value(true));
+        ap(co.fp_to_double,
```

Comment:
This should go under `verify` not `compile` as its only used with `verify` command,